tempy/places.py

```diff
@@ -33,21 +33,21 @@ def _check_container_siblings(cls, container, start=-1, stop=2):
 class Near(Sibling):
     """Check if a TempyREPR object's container if near a certain Tempy Tag"""
 
-    def _reprscore_container_siblings(self, container, child):
+    def _reprscore_container_siblings(self, container):
         return self._check_container_siblings(container)
 
 
 class Before(Sibling):
     """Check if a TempyREPR object's container if before a certain Tempy Tag"""
 
-    def _reprscore_container_following(self, container, child):
+    def _reprscore_container_following(self, container):
         return self._check_container_siblings(container, start=0)
 
 
 class After(Sibling):
     """Check if a TempyREPR object's container if after a certain Tempy Tag"""
 
-    def _reprscore_container_previous(self, container, child):
+    def _reprscore_container_previous(self, container):
```

Comment:
Hi @jamiemott, those methods needs the `child` parameter for all of them are called with at least 3 arguments.
Removing the paramer will make some unittest fail:

```
======================================================================
ERROR: test_before_places (tests.test_TempyREPR.TestSingleTags)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/federicocerchiari/GitHub/TemPy/tests/test_TempyREPR.py", line 156, in test_before_places
    self.assertEqual(Pre()(A()(inst), Span()).render(), '<pre><a>footest</a><span></span></pre>')
  File "/Users/federicocerchiari/GitHub/TemPy/tempy/elements.py", line 232, in render
    inner = self.render_childs(pretty) if not self._void else ""
  File "/Users/federicocerchiari/GitHub/TemPy/tempy/tempy.py", line 673, in render_childs
    return "".join(self._iter_child_renders(pretty=pretty))
  File "/Users/federicocerchiari/GitHub/TemPy/tempy/tempy.py", line 650, in _iter_child_renders
    yield child.render(pretty=pretty)
  File "/Users/federicocerchiari/GitHub/TemPy/tempy/tags.py", line 87, in render
    return super().render(*args, **kwargs)
  File "/Users/federicocerchiari/GitHub/TemPy/tempy/elements.py", line 232, in render
    inner = self.render_childs(pretty) if not self._void else ""
  File "/Users/federicocerchiari/GitHub/TemPy/tempy/tempy.py", line 673, in render_childs
    return "".join(self._iter_child_renders(pretty=pretty))
  File "/Users/federicocerchiari/GitHub/TemPy/tempy/tempy.py", line 652, in _iter_child_renders
    tempyREPR_cls = self._search_for_view(child)
  File "/Users/federicocerchiari/GitHub/TemPy/tempy/tempyrepr.py", line 50, in _search_for_view
    sorted_reprs = sorted(
  File "/Users/federicocerchiari/GitHub/TemPy/tempy/tempyrepr.py", line 39, in _evaluate_tempy_repr
    score += getattr(parent_cls, scorer, lambda *args: 0)(
TypeError: _reprscore_container_following() takes 2 positional arguments but 3 were given
```

It's against pep8 but needed.